### PR TITLE
Bug 1743493 Import pocket spoc tile ID data

### PR DESCRIPTION
**Pull request by jklukas**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1743493

Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [x] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [x] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/pocket/spoc_tile_ids/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+friendly_name: Sponsored Content (Spoc) Tile IDs
+description: |-
+  Lookup table of the currently registered tile IDs for Pocket sponsored content
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/pocket/spoc_tile_ids/metadata.yaml:3]: @kirill-demtchouk I would appreciate getting your eyes on the `metadata.yaml` files and `schema.yaml` files here to see if you disagree with any of the documentation or field descriptions.

File: sql/moz-fx-data-shared-prod/pocket_derived/spoc_tile_ids_history_v1/schema.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  type: DATE
+  description: Date of the tile ID export
+- mode: NULLABLE
+  name: tile_id
+  type: INTEGER
+  description: Numeric ID for the tile
+- mode: NULLABLE
+  name: type
+  type: STRING
+  description: Type of content (should always be 'spoc')
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/pocket_derived/spoc_tile_ids_history_v1/schema.yaml:13]: Is it true that these should always have type="spoc"?

Comment (kirill-demtchouk) [reply sql/moz-fx-data-shared-prod/pocket_derived/spoc_tile_ids_history_v1/schema.yaml:13]: Yes!

**Review by kirill-demtchouk — APPROVED** [review]
LGTM!

**Review by scholtzan — APPROVED** [review]
👍 lgtm

**Pull request merged**